src/BlockEntities/DispenserEntity.cpp

```diff
@@ -283,7 +283,7 @@ UInt32 cDispenserEntity::SpawnProjectileFromDispenser(int a_BlockX, int a_BlockY
 
 Vector3d cDispenserEntity::GetShootVector(NIBBLETYPE a_Meta)
 {
-	switch (a_Meta & 0x7)
+	switch (a_Meta & ~E_META_DROPSPENSER_ACTIVATED)
```

Comment:
I'd appreciate having a specific name for `~E_META_DROPSPENSER_ACTIVATED`, such as `E_META_DROPSPENSER_FACING_MASK`, this will make the code much easier to understand at a glance.